src/constants.tsx

```diff
@@ -367,3 +369,11 @@ export const FI_PLAN_TYPE = 'FI';
 export type FI_PLAN_TYPE = typeof FI_PLAN_TYPE;
 export const IRS_PLAN_TYPE = 'IRS';
 export type IRS_PLAN_TYPE = typeof IRS_PLAN_TYPE;
+
+//
+
+/** dictates how many jurisdiction ids to pass as filter parameter to the
+ * findJurisdictionById endpoint.
+ */
+export const JURISDICTION_NAMES_CHUNK_SIZE = 17;
+export type JURISDICTION_NAMES_CHUNK_SIZE = typeof JURISDICTION_NAMES_CHUNK_SIZE;
```

Comment:
Is this something that could/should be configured differently for different implementations? If so then should go in `env.ts`, otherwise looks good here.

Reply:
Yeah , its configurable